Source: Anders Ossowicki <arkanoid@exherbo.org>
Upstream: No. http://rt.cpan.org:80/Public/Bug/Display.html?id=36554
Reason: Broken DNS servers resolves perl.bv against expectations

--- t/heuristic.t
+++ t/heuristic.t
@@ -5,7 +5,7 @@ if (-f "OFFLINE") {
    exit;
 }
 
-print "1..15\n";
+print "1..14\n";
 
 use URI::Heuristic qw(uf_urlstr uf_url);
 if (shift) {
@@ -44,47 +44,49 @@ print "ok 5\n";
 if (gethostbyname("www.netscape.com")) {
     # DNS probably work, lets run test 6..8
 
-    $URI::Heuristic::MY_COUNTRY = "bv";
-    print "not " unless uf_urlstr("perl/camel.gif") =~ m,^http://www\.perl\.(com|org)/camel\.gif$,;
-    print "ok 6\n";
+# Some DNS providers (*cough*opendns*cough*) acts like morons and resolves
+# stuff that ought not resolve.
+#    $URI::Heuristic::MY_COUNTRY = "bv";
+#    print "not " unless uf_urlstr("perl/camel.gif") =~ m,^http://www\.perl\.(com|org)/camel\.gif$,;
+#    print "ok 6\n";
 
     $URI::Heuristic::MY_COUNTRY = "uk";
     print "not " unless uf_urlstr("perl/camel.gif") =~ m,^http://www\.perl\.(org|co)\.uk/camel\.gif$,;
-    print "ok 7\n";
+    print "ok 6\n";
    
     $ENV{URL_GUESS_PATTERN} = "www.ACME.org www.ACME.com";
     print "not " unless uf_urlstr("perl") eq "http://www.perl.org";
-    print "ok 8\n";
+    print "ok 7\n";
 
 } else {
     # don't make the inocent worry
-    print "Skipping test 6-8 because DNS does not work\n";
-    for (6..8) { print "ok $_\n"; }
+    print "Skipping test 6-7 because DNS does not work\n";
+    for (6..7) { print "ok $_\n"; }
 
 }
 
 {
 local $ENV{URL_GUESS_PATTERN} = "";
 print "not " unless uf_urlstr("perl") eq "http://perl";
-print "ok 9\n";
+print "ok 8\n";
 
 print "not " unless uf_urlstr("http:80") eq "http:80";
-print "ok 10\n";
+print "ok 9\n";
 
 print "not " unless uf_urlstr("mailto:gisle\@aas.no") eq "mailto:gisle\@aas.no";
-print "ok 11\n";
+print "ok 10\n";
 
 print "not " unless uf_urlstr("gisle\@aas.no") eq "mailto:gisle\@aas.no";
-print "ok 12\n";
+print "ok 11\n";
 
 print "not " unless uf_urlstr("Gisle.Aas\@aas.perl.org") eq "mailto:Gisle.Aas\@aas.perl.org";
-print "ok 13\n";
+print "ok 12\n";
 
 print "not " unless uf_url("gopher.sn.no")->scheme eq "gopher";
-print "ok 14\n";
+print "ok 13\n";
 
 print "not " unless uf_urlstr("123.3.3.3:8080/foo") eq "http://123.3.3.3:8080/foo";
-print "ok 15\n";
+print "ok 14\n";
 }
 
 #
